ReSharper.FSharp/test/data/features/quickFixes/addDiscriminatedUnionAllClauses/Partially complete union match.fs

```diff
@@ -0,0 +1,10 @@
+module Say
+type TestUnion =
+    | FieldlessCase
+    | CaseWithSingleField of int
+    | CaseWithMultipleFields of string * int
```

Comment:
Not really an issue but using less words could probably improve readability (and in turn simplify it too):
```fsharp
type U =
    | A
    | B of int
    | C of string * int
```